packages/wonder-blocks-button/package.json

```diff
@@ -2,9 +2,11 @@
   "name": "wonder-blocks-button",
   "version": "0.0.1",
   "description": "",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "source": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepare": "../../node_modules/.bin/webpack"
```

Comment:
The problem with this is that it means that we'll end up re-compling all of our packages whenever someone changes the branch in webapp, b/c webapp is set up to run `npm install` in `third_party/javascript-khansrc/wonder-blocks`.  The reason we do this is to set up symlinks in node_modules just in case someone wants to make changes to wonder-blocks directly within webapp and test their changes in webapp.

To avoid recompiling more often than needed, we should instead be installing `wonder-blocks` as a node modules into third_party/node_modules and rely on the third_party/package.json to track the correct version.  This means that we'll only recompile wonder-blocks-* when switching branches if the version number differs between the two branches.

Reply:
I've fixed this by changing the root package.json's postinstall script to run `npm install --ignore-scripts` in each sub package which prevents `prepare` from running.